release/release.js

```diff
@@ -47,8 +47,8 @@ const masterBranchDefault = "master2"
 const devBranchDefault = "dev2"
 const docBranchDefault = "gh-pages2"
 const versionReleasedDefault = VERSION
-const nextVersionDefault = "12.1.0"
-const versionCodeReleasedDefault = 83
+const nextVersionDefault = "12.2.0"
+const versionCodeReleasedDefault = 85
```

Comment:
why does it get incremented by two?

Reply:
This is just the default value shown in the prompt but one can type whatever one wants. I don't really have to update this file for every release (and I did not update it for 12.0.1, so 83 was 12.0, 84 12.0.1).

Reply:
I could probably make the script smarter (and have it get the value from the manifest in the android repo then increment it for the next version).